src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -516,12 +519,14 @@ assert cmd = do
       preState <- symvmFromCommand cmd
       let errCodes = fromMaybe defaultPanicCodes (assertions cmd)
       verify preState (maxIterations cmd) rpcinfo (Just $ checkAssertions errCodes) >>= \case
-        Right tree -> do
+        Cex tree -> do
           io $ putStrLn "Assertion violation found."
           showCounterexample preState maybesig
           treeShowing tree
           io $ exitWith (ExitFailure 1)
-        Left tree -> do
+        Timeout _ -> do
+          io $ exitWith (ExitFailure 1)
```

Comment:
```suggestion
          io $ do hPutStr stderr "Solver timeout!"
                       exitWith (ExitFailure 1)
```

Reply:
we already print this in `verify` so I left it out here...